apps/cowswap-frontend/src/modules/ordersTable/containers/OrdersReceiptModal/hooks.ts

```diff
@@ -27,14 +27,14 @@ export function useSelectedOrder(): ParsedOrder | null {
   return order
 }
 
-export function useGetShowRecreateModal(order: ParsedOrder | null): Command | null {
-  const setOrderToRecreate = useSetAlternativeOrder()
+export function useGetShowAlternativeModal(order: ParsedOrder | null): Command | null {
```

Comment:
nit `AlternativeOrderModal` instead of `AlternativeModal`?
although I guess you tried to shorten a bit

Reply:
Not intentional, will update